sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/metadata.yaml

```diff
@@ -15,7 +15,6 @@ labels:
   application: firefox
   schedule: daily
 scheduling:
-  allow_field_addition_on_date: '2021-04-19'
```

Comment:
We should remove this, just in case we ever run a backfill. Then the schema will change again.